models/userDao.js

```diff
@@ -35,8 +35,59 @@ const getUserByEmail = async (email) => {
   return result[0]
 }
 
+const getUserById = async (id) => {
+  const result = await appDataSource.query(`
+    SELECT
+      id,
+      name, 
+      password, email
+      email, 
+      phone_number, 
+      address, 
+      birth, 
+      gender,
+      point
+    FROM users
+    WHERE id=?`, [id]
+  )
+  return result[0]
+}
+
+
+const updateUser = async ( userId, password, phoneNumber, address ) => {
+  try {
+    await appDataSource.query(`
+    UPDATE
+      users
+    SET
+      password=?,
+      phone_number=?,
+      address=?
+    WHERE id=?
+    `,
+    [ password, phoneNumber, address, userId]
+  );
+
+    const result = await appDataSource.query(
+      `SELECT *
+        FROM users
+        WHERE id=?
+      `,
+      [userId]
+    );
+
+    return result;
+  }catch(err) {
+    const error = new Error('KEY ERROR')
+    error.statusCode = 500;
```

Comment:
상태코드 500은 예외처리가 안 된 것을 표현 할 때 사용합니다. 400번대 상태 코드로 변경해 주세요.
```suggestion
    error.statusCode = 400;
```